src/app/forms/templates/psv/psv-tech-record.template.ts

```diff
@@ -9,33 +16,34 @@ export const PsvTechRecord: FormNode = {
       name: 'vehicleType',
       label: 'Vehicle type',
       value: '',
-
       type: FormNodeTypes.CONTROL,
       viewType: FormNodeViewTypes.VEHICLETYPE
     },
     {
       name: 'regnDate',
       label: 'Date of first registration',
       value: '',
-
       type: FormNodeTypes.CONTROL,
-      viewType: FormNodeViewTypes.DATE
+      viewType: FormNodeViewTypes.DATE,
+      editType: FormNodeEditTypes.DATE,
+      validators: [{ name: ValidatorNames.Required }],
+      isoDate: false
     },
     {
       name: 'manufactureYear',
       label: 'Year of manufacture',
       value: '',
-
+      width: FormNodeWidth.XS,
```

Comment:
I'm not sure whether this width is actually working 
![image](https://user-images.githubusercontent.com/92087051/196947906-d3552f79-80b0-4bca-ad02-3fb2e0b9dbb4.png)

Reply:
fixt